bridge/client/app/_components/ktb-webhook-settings/ktb-webhook-settings.component.ts

```diff
@@ -29,6 +31,25 @@ export class KtbWebhookSettingsComponent implements OnInit {
     emptyText:
       'No secrets can be found.<p>Secrets can be configured under the menu entry "Secrets" in the Uniform.</p>',
   };
+  public sendFinishedOverlayConfig: DtOverlayConfig = {
+    pinnable: true,
+    originY: 'center',
+  };
+  public _eventType: string | undefined;
+
+  @Input()
+  set eventType(eventType: string | undefined) {
+    if (this._eventType != eventType) {
+      this._eventType = eventType;
+      if (this._eventType !== 'triggered') {
+        this.getFormControl('sendFinished').setValue(false);
+        this.getFormControl('sendFinished').disable();
+      } else {
+        this.getFormControl('sendFinished').setValue(this._webhook.sendFinished ?? true);
```

Comment:
`?? true` is useless here because neither `this._webhook` nor the `sendFinished` property can be `undefined` or `null`.